src/spring/azext_spring/_validators_enterprise.py

```diff
@@ -101,13 +103,156 @@ def validate_builder_resource(namespace):
+    client = get_client(cmd)
+    try:
+        build = client.build_service.get_build(namespace.resource_group,
+                                               namespace.service,
+                                               DEFAULT_BUILD_SERVICE_NAME,
+                                               namespace.name)
+        if build is not None:
+            raise ClientRequestError('Build {} already exists.'.format(namespace.name))
+    except ResourceNotFoundError:
+        pass
+
+
+def validate_build_update(cmd, namespace):
+    client = get_client(cmd)
+    try:
+        build = client.build_service.get_build(namespace.resource_group,
+                                               namespace.service,
+                                               DEFAULT_BUILD_SERVICE_NAME,
+                                               namespace.name)
+        if namespace.builder is None:
+            namespace.builder = build.properties.builder.split("/")[-1]
+        if namespace.build_cpu is None:
+            namespace.build_cpu = build.properties.resource_requests.cpu
+        if namespace.build_memory is None:
+            namespace.build_memory = build.properties.resource_requests.memory
+        if namespace.build_env is None:
+            namespace.build_env = build.properties.env
+    except ResourceNotFoundError:
+        raise ClientRequestError('Build {} does not exist.'.format(namespace.name))
+
+
+def validate_central_build_instance(cmd, namespace):
+    only_support_enterprise(cmd, namespace)
+    client = get_client(cmd)
+    try:
+        build_service = client.build_service.get_build_service(namespace.resource_group,
+                                                               namespace.service,
+                                                               DEFAULT_BUILD_SERVICE_NAME)
+        if not build_service.properties.container_registry:
+            raise ClientRequestError('The command is only supported when using your own container registry.')
+    except ResourceNotFoundError:
+        raise ClientRequestError('Build Service is not enabled.')
+
+
+def validate_source_path(namespace):
+    arguments = [namespace.artifact_path, namespace.source_path]
+    if all(not x for x in arguments):
+        raise InvalidArgumentValueError('One of --artifact-path, --source-path must be provided.')
+    valued_args = [x for x in arguments if x]
+    if len(valued_args) > 1:
+        raise InvalidArgumentValueError('At most one of --artifact-path, --source-path must be provided.')
+
+
+def validate_artifact_path(namespace):
+    if namespace.disable_validation:
+        telemetry.set_user_fault("jar validation is disabled")
+        return
+    if namespace.artifact_path is None:
+        return
+    if os.path.splitext(namespace.artifact_path)[-1] != "jar":
+        return
+    values = _parse_jar_file(namespace.artifact_path)
+    if values is None:
+        # ignore jar_file check
+        return
```

Comment:
It is suggested to merge these if statements.

Reply:
OK, merge two statements.